website/content/docs/auth/kubernetes.mdx

```diff
@@ -101,9 +106,72 @@ management tool.
     For the complete list of configuration options, please see the [API
     documentation](/api/auth/kubernetes).
 
+### How to work with short-lived Kubernetes tokens
+
+Starting in version [1.21][k8s-1.21-changelog], the Kubernetes
+`BoundServiceAccountTokenVolume` feature defaults to enabled. This means the token
+mounted into containers is short-lived and auto-refreshed by default. It is also
+invalidated when the pod is deleted. These tokens are therefore unsuitable to be
+used as the `token_reviewer_jwt` parameter when configuring Kubernetes auth,
+because Vault stores that value in its own storage and does not automatically
+refresh it. There are a few different ways to configure Kubernetes auth in this
+scenario, each with their own tradeoffs.
+
+[k8s-1.21-changelog]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#api-change-2
+
+#### Use the Vault client's JWT as the reviewer JWT
+
+When configuring Kubernetes auth, you can omit the `token_reviewer_jwt`, and Vault
```

Comment:
If you just omit the `token_reviewer_jwt` and not use the `disable_local_ca_jwt` option, it will simply load in the token from the file system of the Vault server and use this one instead. Can this part be added?

Reply:
Good spot, it does indeed need setting for this scenario when Vault is running in Kubernetes. Fixed.

Reply:
Thank you for fixing this! I have now changed my review to approved (for the sake of it). 😄